ui/mirage/handlers/clients.js

```diff
@@ -63,41 +86,19 @@ function generateNamespaceBlock(idx = 0, isLowerCounts = false, ns) {
     counts: {},
     mounts: {},
   };
-  const mounts = [];
 
-  Array.from(Array(5)).forEach((mount, index) => {
-    const [secretSyncs] = arrayOfCounts(randomBetween(min, max), 1);
-    mounts.push({
-      mount_path: `kvv2-engine-${index}`,
-      counts: {
-        clients: secretSyncs,
-        // TODO test with live backend to confirm entity keys are present (and 0) for kv mounts
-        entity_clients: 0,
-        non_entity_clients: 0,
-        distinct_entities: 0,
-        non_entity_tokens: 0,
-        secret_syncs: secretSyncs,
-      },
-    });
-  });
+  const authClients = randomBetween(min, max);
+  const [non_entity_clients, entity_clients] = arrayOfCounts(authClients, 2);
+  const [secret_syncs] = arrayOfCounts(randomBetween(min, max), 1);
+  const [acme_clients] = arrayOfCounts(randomBetween(min, max), 1);
```

Comment:
Here (and then below in the `mounts` array) is where new clients will be added in the future (in addition to the `CLIENT_TYPES` const)